### PR TITLE
CANS-541, FIT team system information added for intake app

### PR DESCRIPTION
**Pull request by AVandana-OSI**
### Jira Story
https://osi-cwds.atlassian.net/browse/CANS-541

## Description
Added a system information to the intake app.

## Tests
- [ ] I have included unit tests 
- [ ] I have included feature tests 
- [ ] I have included other tests 
- [x] I have NOT included tests 
This a just config of adding the Gem to the application.

## Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I have ran all tests for the project.
- [x] I have ran lint/rubocop check for the changed/new files.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build, to check Lint/Rubocop, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by wmitchell-cambria** [comment]
Here's the CodeClimate output:

```
== Gemfile (1 issue) ==
24: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `system_information` should appear before `yard`. [rubocop]

== Gemfile.lock (1 issue) ==
2: Insecure Source URI found: git://github.com/ca-cwds/system_information.git [bundler-audit]
```

File: config/initializers/system_information.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+SystemInformation.configure do |config|
+  config.application = 'Intake App'
+  config.version = ENV.fetch('APP_VERSION', 'unknown').to_s
+  config.checks =
+    [
+      { name: :redis,
+        url: "redis://#{ENV.fetch('REDIS_HOST', 'localhost')}:"\
+          "#{ENV.fetch('REDIS_PORT', 6379)}" },
+      { name: :perry,
+        url: "#{ENV.fetch('AUTHENTICATION_API_BASE_URL', 'http://localhost/perry')}/"\
```
Comment (wmitchell-cambria) [inline config/initializers/system_information.rb]: I think Intake names this `AUTHENTICATION_URL`.

**Comment by AVandana-OSI** [comment]
retest this please

**Comment by wmitchell-cambria** [comment]
retest this please

File: config/initializers/system_information.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+SystemInformation.configure do |config|
+  config.application = 'Intake App'
+  config.version = ENV.fetch('APP_VERSION', 'unknown').to_s
+  config.checks =
+    [
+      { name: :redis,
+        url: "redis://#{ENV.fetch('REDIS_HOST', 'localhost')}:"\
+          "#{ENV.fetch('REDIS_PORT', 6379)}" },
+      { name: :perry,
+        url: "#{ENV.fetch('
```
Comment (edgibbs) [inline config/initializers/system_information.rb]: There's no ENV variable here for Perry?

**Comment by AVandana-OSI** [comment]
retest this please

**Comment by AVandana-OSI** [comment]
retest this please

**Comment by AVandana-OSI** [comment]
retest this please

**Pull request merged**